app/routes/confirm/components/SubmissionConfirmPanel.jsx

```diff
@@ -4,7 +4,7 @@ import EditItemPanel from "../../shared/components/EditItemPanel"
 const mapStateToProps = (state) => {
   return {
     title: `${state.submissions.filter(each => each.selected).length}/${state.submissions.length}`,
-    imagePath: "http://placehold.it/48x48",
+    imagePath: "https://placehold.it/48x48",
```

Comment:
What is this?

Reply:
non-blocking: yeah this could probably be a static image in the repo
![placeholder image example](https://placehold.it/48x48)

Reply:
Fixed this!